components/brave_ads/browser/ads_p2a.h

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_ADS_P2A_H_
+#define BRAVE_COMPONENTS_BRAVE_ADS_BROWSER_ADS_P2A_H_
+
+#include <stdint.h>
```

Comment:
Could we please move this to the implementation as not used in the interface

Reply:
fixed